content/en/docs/releases/kubeflow-1.10.md

```diff
@@ -0,0 +1,295 @@
+        </td>
+      </tr>
+      <tr>
+        <td>Volumes Web App</td>
+        <td>
+          <a href="https://github.com/kubeflow/kubeflow/tree/v1.10.0/components/crud-web-apps/tensorboards">v1.10.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Pipelines Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Pipelines Working Group</td>
+        <td>Kubeflow Pipelines</td>
+        <td>
+          <a href="https://github.com/kubeflow/pipelines/releases/tag/2.4.1">v2.4.1</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Serving Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Serving Working Group</td>
+        <td>KServe</td>
+        <td>
+          <a href="https://github.com/kserve/kserve/releases/tag/v0.14.1">v0.14.1</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Training Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="1" class="align-middle">Training Working Group</td>
+        <td>Training Operator</td>
+        <td>
+          <a href="https://github.com/kubeflow/training-operator/releases/tag/v1.9.0">v1.9.0</a>
+        </td>
+      </tr>
+      <!-- ======================= -->
+      <!-- Data Working Group -->
+      <!-- ======================= -->
+      <tr>
+        <td rowspan="2" class="align-middle">Data Working Group</td>
+        <td>Model Registry</td>
+        <td>
+          <a href="https://github.com/kubeflow/model-registry/releases/tag/v0.2.15">v0.2.15</a>
+        </td>
+      </tr>
+      <tr>
+        <td>Spark Operator</td>
+        <td>
+          <a href="https://github.com/kubeflow/spark-operator/releases/tag/v2.1.0">v2.1.0</a>
+        </td>
+      </tr>
+  </tbody>
+</table>
+</div>
+
+### Dependency Versions (Manifests)
+
+{{% alert title="Note" color="warning" %}}
+This information is only for the manifests found in the <a href="https://github.com/kubeflow/manifests">kubeflow/manifests</a> repository, packaged distributions may have different requirements or supported versions.
+{{% /alert %}}
+
+<div class="table-responsive">
+<table class="table table-bordered">
+    <thead class="thead-light">
+      <tr>
+        <th>Dependency</th>
+        <th>Validated or Included Version(s)</th>
+        <th>Notes</th>
+      </tr>
+    </thead>
+  <tbody>
+      <!-- ======================= -->
+      <!-- Kubernetes -->
+      <!-- ======================= -->
+      <tr>
+        <td>
+          <a href="https://kubernetes.io/">Kubernetes</a>
+        </td>
+        <td>1.32</td>
+        <td rowspan="4" class="align-middle">
+          <i>Other versions may work, but have not been validated by the <a href="https://github.com/kubeflow/community/tree/master/wg-manifests">Kubeflow Manifests Working Group</a>.</i>
```

Comment:
We support 1.31-1.33 and it should work on 1.29+

Reply:
usually we add one base version. Should I add 1.33?

Reply:
We should list whatever version we actually tested on the manifests repo:

> Other versions may work, but have not been validated by the Kubeflow Manifests Working Group